modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/views/stories/StoryTopFivePodcastsView.kt

```diff
@@ -1,42 +1,166 @@
 ) {
     Column(
-        verticalArrangement = Arrangement.Center,
-        modifier = modifier.verticalScroll(rememberScrollState()),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = modifier
+            .fillMaxSize()
+            .podcastDynamicBackground(story.topPodcasts[0].toPodcast())
+            .verticalScroll(rememberScrollState())
     ) {
-        TextH30(
-            text = "Your Top Podcasts",
-            textAlign = TextAlign.Center,
-            color = story.tintColor,
-            modifier = modifier
-                .fillMaxWidth()
-                .padding(vertical = 16.dp)
+        Spacer(modifier = modifier.height(40.dp))
+
+        Spacer(modifier = modifier.weight(1f))
+
+        Title(story)
+
+        Spacer(modifier = modifier.weight(0.5f))
+
+        PodcastList(story)
+
+        Spacer(modifier = modifier.weight(1f))
+
+        PodcastLogoWhite()
+
+        Spacer(modifier = modifier.height(40.dp))
+    }
+}
+
+@Composable
+private fun Title(
+    story: StoryTopFivePodcasts,
+    modifier: Modifier = Modifier,
+) {
+    val text = stringResource(R.string.end_of_year_story_top_podcasts)
+    Text(
+        text = text,
+        textAlign = TextAlign.Center,
+        color = story.tintColor,
+        fontSize = 20.sp,
+        fontWeight = FontWeight.SemiBold,
+        modifier = modifier
+            .padding(horizontal = 40.dp)
+            .fillMaxWidth()
+    )
+}
+
+@Composable
+private fun PodcastList(story: StoryTopFivePodcasts) {
+    story.topPodcasts.forEachIndexed { index, topPodcast ->
+        PodcastItem(
+            podcast = topPodcast.toPodcast(),
+            position = index,
+            tintColor = story.tintColor
+        )
+    }
+}
+
+@Composable
+fun PodcastItem(
+    podcast: Podcast,
+    position: Int,
+    tintColor: Color,
+    modifier: Modifier = Modifier,
+) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(horizontal = 40.dp)
+    ) {
+        TextH20(
+            text = "${position + 1}.",
+            color = tintColor,
+            modifier = modifier.padding(end = 14.dp)
         )
-        story.topPodcasts.forEach { topPodcast ->
+        Row(
+            modifier = modifier
+                .padding(vertical = 10.dp)
+                .weight(1f),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            PodcastImage(
+                uuid = podcast.uuid,
+                modifier = modifier.size(64.dp)
+            )
+            Column(
+                modifier = modifier
+                    .padding(start = 14.dp)
+            ) {
+                TextH20(
+                    text = podcast.title,
+                    color = tintColor,
+                    maxLines = 2,
+                    modifier = modifier
+                        .padding(bottom = 3.dp)
+                )
```

Comment:
```suggestion
                TextH30(
                    text = podcast.title,
                    color = tintColor,
                    maxLines = 2,
                    fontWeight = FontWeight.Bold,
                    modifier = modifier
                        .padding(bottom = 3.dp)
                )
```

| Before | After |
| --- | --- |
| ![Screenshot_20221114_115929](https://user-images.githubusercontent.com/308331/201557234-c5928388-5491-4948-95a1-8958182bf850.png) | ![Screenshot_20221114_115520](https://user-images.githubusercontent.com/308331/201557243-6feb57a8-3036-422e-931e-58a7353b74ee.png) |